services/urls.py

```diff
@@ -57,6 +58,16 @@ def get_dashboard_base_url() -> str:
         return configured_dashboard_url or "https://app.codecov.io"
 
 
+def _get_username_for_url(repository: Repository) -> str:
+    username = repository.owner.username
+    if repository.owner.service == Service.GITLAB.value:
```

Comment:
Idk if we save service always as "gitlab" vs "GitLab" or something like that, might be worth lowercasing this just in case